model/shared/src/main/scala/explore/model/SpectroscopyConfigurationOptions.scala

```diff
@@ -4,36 +4,52 @@
 package explore.model
 
 import cats.Eq
+import coulomb.Quantity
+import coulomb.cats.implicits._
 import eu.timepit.refined.cats._
 import eu.timepit.refined.types.numeric.PosBigDecimal
 import eu.timepit.refined.types.numeric.PosInt
 import explore.model.enum.SpectroscopyCapabilities
 import explore.modes.FocalPlane
 import lucuma.core.math.Angle
 import lucuma.core.math.Wavelength
+import lucuma.core.math.units.Micrometer
 import monocle.macros.Lenses
 
+/**
+ * We want to store wavelengths in units to allow 0 in the UI
+ * but later we convert these units for calculations were Wavelength 0 is not allowed
```

Comment:
We need to allow 0 in the UI as a temporary state?

Reply:
Yes, when you start typing e.g. 0.2, the input tries to convert the first 0 into a 0 wavelength and it fails thus forbidding the 0 to be entered which is not very good
In this model 0 is allowed but ignored until a valid wavelength is entered
This is noticeable here because we are using microns. With nanometers you most likely wouldn't see it